container/navigation.go

```diff
@@ -0,0 +1,196 @@
+package container
+
+import (
+	"fyne.io/fyne/v2"
+	"fyne.io/fyne/v2/layout"
+	"fyne.io/fyne/v2/theme"
+	"fyne.io/fyne/v2/widget"
+)
+
+// Navigation container is used to provide your application with a control bar and an area for content objects.
+// Objects can be any CanvasObject, and only the most recent one will be visible.
+//
+// Since: 2.6
+type Navigation struct {
+	widget.BaseWidget
+
+	Root  fyne.CanvasObject
+	Title string
+	Back  NavigationObject
```

Comment:
I'm not a fan of exporting these objects either - a standard back and forward buttons should be sufficient